src/script/common/c_converter.h

```diff
@@ -120,7 +120,8 @@ void                push_aabb3f_vector  (lua_State *L, const std::vector<aabb3f>
 
 void                warn_if_field_exists(lua_State *L, int table,
                                          const char *fieldname,
-                                         const std::string &message);
+                                         const std::string_view name,
+                                         const std::string_view message);
```

Comment:
btw there's no need to mark a string_view as const

Reply:
Fxied in https://github.com/minetest/minetest/pull/14794/commits/9584db2a67e102c37d39c7dc1ef17e9867d8c5c8